mlflow/pyfunc/__init__.py

```diff
@@ -127,13 +136,28 @@ def load_pyfunc(path, run_id=None):
         raise Exception("Format '{format}' not found not in {path}.".format(format=FLAVOR_NAME,
                                                                             path=conf_path))
     conf = model.flavors[FLAVOR_NAME]
+    model_py_version = conf.get(PY_VERSION) if PY_VERSION in conf else None
+    if not suppress_warnings:
+        _warn_potentially_incompatible_py_version_if_necessary(model_py_version=model_py_version)
     if CODE in conf and conf[CODE]:
         code_path = os.path.join(path, conf[CODE])
         sys.path = [code_path] + _get_code_dirs(code_path) + sys.path
     data_path = os.path.join(path, conf[DATA]) if (DATA in conf) else path
     return importlib.import_module(conf[MAIN]).load_pyfunc(data_path)
 
 
+def _warn_potentially_incompatible_py_version_if_necessary(model_py_version):
+    if model_py_version is None:
+        print("The specified model does not have a specified Python version. It may be incompatible"
```

Comment:
These should use eprint, `from mlflow.utils.logging_utils import eprint`, to ensure we print to stderr instead of stdout.

Reply:
Thanks, using `eprint` now!